crates/pgsrv/src/handler.rs

```diff
@@ -252,6 +252,7 @@ impl ProtocolHandler {
             .set_and_log(Some(max_tunnel_count), VarSetter::System);
         vars.max_credentials_count
             .set_and_log(Some(max_credentials_count), VarSetter::System);
+        vars.is_cloud_instance.set_and_log(true, VarSetter::System);
```

Comment:
So this will set the variable to true when running as a server (e.g. via `glaredb server`). Our slts connect to this since it'll spin up an in-process server, then use the postgres client to connect.

However, we'll be able to tell if the connection has been proxied through the existence of some parameters that get set by pgsrv. For example, we could do something like this:

```rust
// Determine if we're running as a cloud instance by trying to 
// read a non-standard parameter that gets set by pgsrv.
let is_cloud_instance = params.get(&GLAREDB_DATABASE_ID_KEY.key).is_ok();
```

This would require that we make the `key` field public.

Reply:
I _think_ that should work